src/gamestate/system_state.cpp

```diff
@@ -166,6 +171,34 @@ void state::render() { // called to render the frame may (and should) delay retu
 		}
 	}
 
+
+	if(!mouse_probe.under_mouse) {
+		if(ui_state.last_tooltip != ui_state.map_tooltip_window) {
+			ui_state.last_tooltip = ui_state.map_tooltip_window;
+		} 
+		if(ui_state.last_tooltip && ui_state.tooltip->is_visible()) {
```

Comment:
So what this evaluates is whether a tooltip had been created previously, and if so, whether the ui element that created it is still visible. Note that this is within a black that will only run if nothing is under the mouse, so I am not sure what the intent is

Reply:
copy and paste, need to fix ig